### ISSUE TITLE
[llvm-cov][MC/DC] Output should list *all* "relevant" test vectors

### ISSUE THREAD

**Issue by Swatinem** [llvm/llvm-project#105679]
Right now, the output for MC/DC only shows the *executed* test vectors, it looks a bit like this (cleaned up):

```
  ------------------
  |  Executed MC/DC Test Vectors:
  |
  |     C1, C2, C3, C4    Result
  |  1 { F,  F,  -,  -  = F      }
  |  2 { F,  T,  F,  F  = F      }
  |  3 { F,  T,  T,  -  = T      }
  |  4 { F,  T,  F,  T  = T      }
  |
  |  C1-Pair: not covered
  |  C2-Pair: covered: (1,3)
  |  C3-Pair: covered: (2,3)
  |  C4-Pair: covered: (2,4)
  |  MC/DC Coverage for Decision: 75.00%
  ------------------
```

It would be nice to show *all* the test vectors (the "relevant" ones).
From a developers point of view, I want to see at first glance what I am missing, so I know which test cases I have to add to my test suite to improve the outcome of MC/DC.

---

Working on https://github.com/llvm/llvm-project/pull/105511, I noticed that this might be a bit more complex to implement, as it would seem that this data is not readily available internally, as opposed to only not being displayed/exported.

**Comment by evodius96** [comment]
I agree this would be a useful extension.  Off hand, I actually don't think it would be that difficult.  Before marking executed test vectors, the algorithm in `CoverageMapping.cpp` used by `llvm-cov` enumerates all possible test vectors.  You'd only need to make sure they were also visualized.  I think the reason why I opted not to show this was because the number of possible test vectors can be large as the expressions grow.  If someone wanted to implement this, I would recommend tying it to a command line option.
